src/modules/feedback/i18n/feedback.provider.js

```diff
@@ -12,7 +12,8 @@ export const provide = (lang) => {
 				mapFeedback_privacyPolicy: 'Privacy Policy',
 				mapFeedback_pleaseSelect: 'Please draw at least one Point to mark the location of your feedback',
 				mapFeedback_disclaimer:
-					'In some cases, the LDBV cannot adopt your feedback. For questions relating to your feedback and to keep you informed about the work in progress we recommend submitting your email address as well.'
+					'In some cases, the LDBV cannot adopt your feedback. For questions relating to your feedback and to keep you informed about the work in progress we recommend submitting your email address as well.',
+				mapFeedback_geometry_missing: 'Create a geometry for your feedback'
```

Comment:
`mapFeedback_pleaseSelect` can be removed, but 
`mapFeedback_geometry_missing` should have the value of `mapFeedback_pleaseSelect`